tools/migration/db/util/mysql_pgsql_1_5_0.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source $PWD/db/util/pgsql.sh
+source $PWD/db/util/alembic.sh
+
+DBCNF="-hlocalhost -u${DB_USR}"
+
+function mysql_2_pgsql_1_5_0 {
+    
+    alembic_up mysql '1.5.0'
```

Comment:
what if this fails?

Reply:
the shell will be exited with non-zero code as "set -e"

Reply:
will add set -e to each of these shell scripts.